### PR TITLE
[Synapse] Update synapse's release history

### PR DESCRIPTION
**Pull request by aim-for-better**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/synapse/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,14 @@
...
 ===============
 
+0.2.0
+++++++
+* Supported resource id parameter when getting, updating or deleting synapse Workspace, Spark pool and SQL pool.
+* Added command group `az synapse workspace firewall-rule` to add and delete firewall rule.
+* Added a command `az syanpse workspace check-name` to check a workspace exist or not.
+* Refactored the parameters'name under command group `az synapse spark` and `az synapse sql`.
+
+
```
Comment (idear1203) [inline src/synapse/HISTORY.rst:13]: Remove one empty line?

**Comment by yonzhan** [comment]
Synapse

File: src/synapse/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+0.2.0
```
Comment (jsntcy) [inline src/synapse/HISTORY.rst:6]: You need history for 0.3.0

File: src/synapse/HISTORY.rst (hunk 1 above)
Comment (aim-for-better) [reply src/synapse/HISTORY.rst:16]: > Remove one empty line?

Other service also has one empty line. Such as azure-firewall

**Comment by haroldrandom** [comment]
Could you rebase the master and try again?

**Pull request merged**